action.yaml

```diff
@@ -308,8 +308,13 @@ runs:
         echo "Files changed: $files"
         DBT_DEPLOY=false
 
+        root_folder="${{ inputs.root-folder }}"
+        if [[ $root_folder == "./" ]]; then
+          root_folder=""
+        fi
+
         for file in $files; do
-          if [[ $file =~ ^"${{ inputs.root-folder }}".* ]]; then
+          if [[ $file =~ ^"$root_folder".* ]]; then
```

Comment:
Do we also need some kind of validation that the root folder does not start with a forward slash, which presumably would not match with any `file`?

Reply:
issue to track the change: https://github.com/astronomer/deploy-action/issues/85

Reply:
Currently we run `astronomer/deploy-action@v0.7` prior to `astro deployment variable update` in CI. If there are no changes, this step fails. Will set the `root-folder` to see if that gets us moving forward. Ideally these changes could be versioned, if possible, though.

Reply:
yup, since this is a fix, we are planning to release a `v0.7.1` with these PR changes